WordPress/gutenberg
# Components: Do not render FormTokenField label when not defined

## What?


Changes introduced this PR doesn't not rendering the label elements when the property value is not defined

## Why?

Because currently, the component renders element with empty content

## How?

It will render the label components only when the property is defined

## Testing Instructions





Using storybook

1. Run storybook locally

```
npm run storybook:dev
```

2. Select the default story for the TokenFormField component

`http://localhost:50240/?path=/story/components-formtokenfield--default&args=label:`

3. Confirm that when the label is empty or not defined, now the component does not add an additional empty space

before (trunk) | after (this PR)
--------|------
<img width="861" alt="Screenshot 2024-05-02 at 19 46 16" src="https://github.com/WordPress/gutenberg/assets/77539/3cb8a81f-33b7-4eb4-8d5d-b7dcf0da06fd"> | <img width="867" alt="Screenshot 2024-05-02 at 19 43 42" src="https://github.com/WordPress/gutenberg/assets/77539/7cfbcf52-9b67-46c0-8caf-e79c9feb1122">

4. You can confirm that currently (trunk) the component renders en empty `label` attribute

<img width="390" alt="image" src="https://github.com/WordPress/gutenberg/assets/77539/c8a1d4a9-2db4-4e5f-9386-40977367b2cf">


### Testing Instructions for Keyboard


## Screenshots or screencast
